index.html

```diff
@@ -356,23 +403,21 @@ <h3> Normative Specifications </h3>
                 Group before being considered in the Working Group.
               </p>
             </dd>
-            <dt> <a href="https://webscreens.github.io/openscreenprotocol/">Open Screen Protocol</a> </dt>
+            <dt> <a href="https://w3c.github.io/openscreenprotocol/">Open Screen Protocol</a> </dt>
             <dd>
               <p>
                 A suite of network protocols that allow user agents to implement
                 the Presentation API and the Remote Playback API in an
                 interoperable fashion for browsers and presentation displays
                 connected via the same local area network.
               </p>
-              <p class="draft-status"> <b>Draft state:</b> <a href="https://webscreens.github.io/openscreenprotocol/">
+              <p class="draft-status"> <b>First Public Working Draft</b> <a href="https://www.w3.org/TR/2021/WD-openscreenprotocol-20210318/">
                   Adopted from the Second Screen Community Group</a></p>
               <p class="milestone">
-                <b>Expected completion:</b> Q2 2021
+                <b>Expected completion:</b> Q4 2023
               </p>
             </dd>
           </dl>
-          <p> The Working Group may decide to group the API and protocol suite
```

Comment:
@tidoust is this explicit clause needed to give the WG this flexibility? If not needed, removing it is fine.

Reply:
I remember we added it to make sure that the WG would have the flexibility to shuffle specs around as needed. I would apply the "if it ain't broke, don't fix it" rule of thumb here. Is removal of that sentence meant to solve a practical issue?

Reply:
The group adopted the OSP specification as a whole, but I suppose we could decide to split it up in the future.  I'll restore this sentence.